### PR TITLE
Minor: Add OpenAPI 3.0 support for serverless downloadDocumentation command

### PR DESCRIPTION
**Pull request by rovellipaolo**
The `serverless downloadDocumentation` command currently supports to download the documentation only in OpenAPI 2.0 (Swagger).
This pull request adds support to (optionally) download the documentation also in OpenAPI 3.0.

Example:
```
$ serverless downloadDocumentation --outputFileName=filename.ext --exportType oas30
```

See also: https://github.com/failsafe-engineering/serverless-aws-apigateway-documentation/issues/40

### CODE REVIEW

File: src/downloadDocumentation.js
Comment (rovellipaolo) [inline src/downloadDocumentation.js:69]: For  more details see: https://docs.aws.amazon.com/cli/latest/reference/apigateway/get-export.html
> --export-type (string)
> The type of export. Acceptable values are 'oas30' for OpenAPI 3.0.x and 'swagger' for Swagger/OpenAPI 2.0.

And also: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-export-api.html
> With the Export API, you export an existing REST API by submitting a GET request, specifying the to-be-exported API as part of URL paths. The request URL is of the following format:
> - OpenAPI 3.0
>    https://<host>/restapis/<restapi_id>/stages/<stage_name>/exports/oas30
> - OpenAPI 2.0
>    https://<host>/restapis/<restapi_id>/stages/<stage_name>/exports/swagger

File: src/downloadDocumentation.spec.js
Comment (rovellipaolo) [inline src/downloadDocumentation.spec.js]: This change is totally unrelated. But, as I was touching this test suite, I thought it useful to add coverage also for this already existing use case (i.e., using a `yaml` file extension).
If you prefer not to, let me know and I will revert it.

**Comment by ottopecz** [comment]
Hi @rovellipaolo - Thanks for the PR. We're looking at it... In the meantime... what's the "weight" of the change in your head? "patch", "minor", "major"?

**Comment by rovellipaolo** [comment]
> Hi @rovellipaolo - Thanks for the PR. We're looking at it... In the meantime... what's the "weight" of the change in your head? "patch", "minor", "major"?

I would say it's "minor". It adds a new, small functionality and it's backward-compatible.

File: src/downloadDocumentation.spec.js
Comment (ottopecz) [inline src/downloadDocumentation.spec.js]: Not sure I understand why it's necessary to mix up async/await with the promise API... ? It should work without the `.then`

File: src/downloadDocumentation.spec.js
Comment (ottopecz) [inline src/downloadDocumentation.spec.js]: See above.

**Comment by ottopecz** [comment]
@rovellipaolo - I added two small review comments and I updated the the title of the PR.

File: src/downloadDocumentation.spec.js
Comment (rovellipaolo) [reply src/downloadDocumentation.spec.js]: You're totally right! I've copy-pasted the structure from [another previously existing test](https://github.com/failsafe-engineering/serverless-aws-apigateway-documentation/blob/3379d9144277dbc69ba6f907c49f4083ecf8d1e4/src/downloadDocumentation.spec.js#L48), for both simplicity and consistency with the rest of the code, but somehow I did not notice this.

Fixed in all the tests of this test suite: https://github.com/failsafe-engineering/serverless-aws-apigateway-documentation/pull/39/commits/4767337d00d357618c556befad50a9a875b3db1b

**Comment by rovellipaolo** [comment]
> @rovellipaolo - I added two small review comments and I updated the the title of the PR.

Thanks! I've made the requested changes. Please, have a look at them.

**Comment by ottopecz** [comment]
@rovellipaolo - I approved. I'll merge it when it goes through... Once it's been merged the pipeline should publish it as a minor...

**Review by ottopecz — APPROVED** [review]

**Comment by ottopecz** [comment]
@rovellipaolo - It looks like we have some pipeline issues... I'll fix it and publish the new version.

**Comment by ottopecz** [comment]
@rovellipaolo - I had to create a new [PR](https://github.com/failsafe-engineering/serverless-aws-apigateway-documentation/pull/41) because the pipeline can't deal with forks. 🤦‍♂️

Anyway, I's been out with [v1.2.0](https://www.npmjs.com/package/@kakkuk/serverless-aws-apigateway-documentation).

**Pull request merged**
